reference/clinic.v1.yaml

```
@@ -316,608 +316,436 @@ paths:
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=5'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.averageGlucoseMmol
-          description: Average glucose value of records
+          description: Average glucose value of records in Mmol/L
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInVeryLowPercent
-          description: Percentage of time below 54 mg/dL
+          description: Percentage of time [0.0-1.0]  below 54 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInAnyLowPercent
-          description: Percentage of time below 70 mg/dL
+          description: Percentage of time [0.0-1.0]  below 70 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInLowPercent
-          description: Percentage of time in range 54-70 mg/dL
+          description: Percentage of time [0.0-1.0]  in range 54-70 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInTargetPercent
-          description: Percentage of time in range 70-180 mg/dL
+          description: Percentage of time [0.0-1.0]  in range 70-180 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInHighPercent
-          description: Percentage of time in range 180-250 mg/dL
+          description: Percentage of time [0.0-1.0]  in range 180-250 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInVeryHighPercent
-          description: Percentage of time above 250 mg/dL
+          description: Percentage of time [0.0-1.0]  above 250 mg/dL
         - schema:
-            type: string
-            pattern: ^(>=|>|<=|<)\d\.\d\d?$
-            example: '>=0.7'
+            $ref: '#/components/schemas/FloatFilter'
           in: query
           name: bgm.timeInExtremeHighPercent
-          description: Percentage of time above 350 mg/dL
+          description: Percentage of time [0.0-1.0]  above 350 mg/dL
```

```suggestion
          description: Percentage of time [0.0-1.0] above 350 mg/dL
```
Consider removing the extra space the next time you're making changes here.